mrt/static/js/printouts.js

```diff
@@ -22,12 +22,10 @@ $(function () {
         job_row.addClass('success');
         job_row.find('.status').text(STATUS.FINISHED);
         
-        var filePath = "[?:[a-zA-Z0-9-_\.]+(?:.xls|.xlsx)";
-        
-        var html = 'Success';
-        if (result.toString().match(filePath)) {
+        html = result;
```

Comment:
By removing `var` here the variable is hoisted to the global scope and that's generally bad practice.